### ISSUE TITLE
Unlocking an exisiting keychain

### ISSUE THREAD

**Issue by xfreebird** [KrauseFx/fastlane#566]
Hi,

I've been looking through documentation and I haven't seen a way to unlock a specified keychain.
To unlock the keychain a password is required.

Is this action available ?

Usually on CI server the keychain gets locked, especially when the session is run through ssh (despite the fact that the ssh daemon runs in the user ui session).

Any pointers would help.

Cheers,
Nicolae

**Comment by KrauseFx** [comment]
I haven't done this myself, but basically you can use the shell command to unlock your Keychain

From https://github.com/KrauseFx/fastlane/blob/80c57b452052e3f89dc646747fac4ccf542e8058/lib/fastlane/actions/create_keychain.rb#L22


```ruby
sh("security unlock-keychain -p #{escaped_password} #{escaped_name}", log: false)
```

**Comment by KrauseFx** [comment]
You can just add this to your `Fastfile`

**Comment by xfreebird** [comment]
Thanks @KrauseFx, I am aware of the command line option.
Since there are several existing keychain fastlane actions, 
I think it would be great to have and this one in place.

Cheers,
Nicolae

**Comment by xfreebird** [comment]
This was implemented with #580.

**Comment by KrauseFx** [comment]
Great, thanks again for your contribution :+1:
